### PR TITLE
COMMERCE-7159 - Fix Wrong layout in Diagram tabIX Wrong layout in Diagram tab

### PR DESCRIPTION
**Pull request by garofalog**
result before fix

_thumb_421766

result after fix

_thumb_421765

### CODE REVIEW

**Comment by john-co** [comment]
Test failure looks unrelated, test is failing on random error: https://issues.liferay.com/browse/LRQA-69822

cc/ @vicnate5

**Comment by john-co** [comment]
ci:test:relevant

**Comment by john-co** [comment]
Reviewed. Test failures are unrelated. :heavy_check_mark:

**Review by kresimir-coko — APPROVED** [review]
Looks good to me

**Review by markocikos — APPROVED** [review]
LGTM!

**Comment by markocikos** [comment]
ci:forward

**Comment by john-co** [comment]
ci:forward

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.diagram-fixed-row {
+    height: 230px !important;
```
Comment (markocikos) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/css/main.scss:2]: Do we need `!important`? It should be used as a last resort, to overwrite inline JS, added by third party JS. If you're just overriding simple CSS, add it along with with `.diagram-fixed-row`. For example, if the style is on `.row`, add `.diagram-fixed-row.row {...}`

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/edit_cp_definition_diagram_setting.jsp [hunk 2]
```diff
@@ -28,6 +28,8 @@ String type = BeanParamUtil.getString(cpDefinitionDiagramSetting, renderRequest,
 CPDefinitionDiagramType cpDefinitionDiagramType = cpDefinitionDiagramSettingDisplayContext.getCPDefinitionDiagramType(type);
 %>
 
+<link href="<%= PortalUtil.getStaticResourceURL(request, PortalUtil.getPathModule() + "/commerce-shop-by-diagram-web/css/main.css") %>" rel="stylesheet" />
+
```
Comment (markocikos) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/edit_cp_definition_diagram_setting.jsp]: Why is a `link` tag here? 

(1) Adding a `link` tag in `body` is a bad practice. It means page will re-render twice, causing performance and FOUC issues.
(2) In Liferay we have standard ways to include CSS files, and this is not one of them. A module-scoped CSS file is most commonly added through [com.liferay.portlet.header-portlet-css property](https://github.com/liferay/liferay-portal/blob/master/modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/AccountEntriesAdminPortlet.java#L38).

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/edit_cp_definition_diagram_setting.jsp [hunk 3]
```diff
@@ -98,22 +90,40 @@ CPDefinitionDiagramType cpDefinitionDiagramType = cpDefinitionDiagramSettingDisp
...
-						itemsPerPage="<%= 10 %>"
-						namespace="<%= liferayPortletResponse.getNamespace() %>"
-						pageNumber="<%= 1 %>"
-						portletURL="<%= cpDefinitionDiagramSettingDisplayContext.getPortletURL() %>"
-						style="stacked"
-					/>
-				</commerce-ui:panel>
+		<div class="row">
+			<div class="col-lg-8">
+
```
Comment (markocikos) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/edit_cp_definition_diagram_setting.jsp:98]: Blank row here is probably what caused CI SF failure.

**Review by markocikos — CHANGES_REQUESTED** [review]
@garofalog It's a shame you ninja-added a commit. That stopped forwarding that would have been successful (see green CI report). I left some comments on the added commit. I'd consider the `link` tag issue a blocker, unless there is a very good reason for it that I'm not seeing.

**Comment by garofalog** [comment]
Hello @markocikos, this pr was approved before yesterday. Then I made some other changes waiting to talk to other team members to fix another doubt. This change had to be in this pr, but i can't change a pr in a draft pr. I know you are reviewing this  constantly but we have several changes on-the-go and i know sometimes it's difficult to understand what we are doing. I'll reopen a new draft and resend it when all will be fixed

**Pull request closed without merging**